CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+## Contributing Guidelines
+
+So you have made your mind to contribute to this project? Here's everything you'll need to make your first contribution.
+
+### What can I help with?
+
+We welcome contributions of all sorts. Even the most minor changes are warmly welcomed. You may choose to work on an existing issue or propose your own.
+
+Some examples of contributions are:
+
+1. Documentation: From spell corrections to documenting the necessary information, all comes in this section.
+
+2. Bugs: Any bug fixes are counted in this section. If you see a new bug, feel free to create a new issue mentioning the problem faced.
+
+3. Design: You may like to propose some design changes for the website. If you wish to do so, create an issue mentioning the changes you want to add, like screenshots, Figma designs, or relevant information.
+
+4. Feature Requests: If you would like to see some new features around the website, you can always create a new issue mentioning the requested feature.
+
+**Important:** Every PR must correspond to an issue. If there isn't an issue related to the PR, open a new issue in the Issues section.
+
+### Grabbing an issue
+
+To work on an existing issue, comment on it and say you're working on that issue. Any member will assign you to the issue. This is to avoid conflicts with others also working on the issue.
+
+You can always seek help or discuss on the issue page.
+
+### How to Contribute?
+
+1. Fork this repository and set up the development environment as mentioned [here](README.md).
+
+2. Check out to a new branch for the patch.
+
+```bash
+git checkout -b <branch name>
+```
+
+3. Write your code.
+
+4. Before committing changes, you should verify if the issue is fixed and also run the following command to format the code.
+
+```bash
+npm run format
+```
+
+5. Commit your changes. Try to follow the [best practices][best_practices] while committing changes.
+
+```bash
+git commit
```

Comment:
`git commit -s` can be introduced here as we are telling them to use `git commit`

Reply:
Is sign off necessary? :thinking:

Reply:
adds credibility I guess can be followed as a good practice

Reply:
Sure then